Translations/fr.po

```diff
@@ -8657,20 +8662,20 @@ msgid ""
 "and loyal in nature. The curse that fell upon the followers of King Leoric "
 "would fall especially hard upon him."
 msgstr ""
-"Si c'est réellement Lachdanan que vous avez rencontré, alors je vous "
-"conseillerais de l'aider. J'ai eu affaire à lui à plusieurs reprises et je "
-"l'ai trouvé de nature honnête et loyale. La malédiction qui s'est abattu sur "
-"les partisans du roi Leoric a dû être encore plus terrible envers lui."
+"Si c'est bien Lachdanan que vous avez rencontré, je serais d'avis que vous "
+"l'aidiez. J'ai eu affaire à lui à plusieurs reprises et j'ai vu en lui un "
+"homme honnête et loyal. La malédiction qui frappe les compagnons du roi "
+"Leoric doit être encore plus terrible pour lui."
 
 #. TRANSLATORS: Quest dialog spoken by Farnham
 #: Source/textdat.cpp:166
 msgid ""
 " Lachdanan is dead. Everybody knows that, and you can't fool me into "
 "thinking any other way. You can't talk to the dead. I know!"
 msgstr ""
-" Lachdanan est mort. Tout le monde le sait, et vous ne pouvez pas me tromper "
-"en me faisant penser autrement. Vous ne pouvez pas parler aux morts. Je le "
-"sais!"
+"Lachdanan est mort. Tout le monde sait ça, et vous pouvez pas me berner en "
```

Comment:
```suggestion
"Lachdanan est mort. Tout le monde sait ça, et vous ne pouvez pas me berner en "
```

Reply:
I don’t agree on that. It’s Farnham, and I doubt he cares so much about his language level. The use of "ne" should be, in my opinion, very occasional.